packages/components/src/index.ts

```diff
@@ -6,8 +6,8 @@ export * from "./components/Navigation";
 export * from "./components/PressableHighlight";
 export * from "./components/EDSProvider";
 export * from "./components/Spacer";
-export * from "./components/ButtonGroup";
-export * from "./components/ToggleButton";
+export * from "./components/Button/ButtonGroup";
+export * from "./components/Button/ToggleButton";
```

Comment:
I think you don't have to export these any more as they are now part of the `Button` export (?)